Tools/autotest/common.py

```diff
@@ -74,6 +73,39 @@ def mavproxy_options(self):
 
         return ret
 
+    def vehicleinfo_key(self):
+        return self.log_name
+
+    def apply_parameters_using_sitl(self):
+        '''start SITL, apply parameter file, stop SITL'''
+        sitl = util.start_SITL(self.binary,
+                               wipe=True,
+                               model=self.frame,
+                               home=self.home,
+                               speedup=self.speedup_default)
+        self.mavproxy = util.start_MAVProxy_SITL(self.log_name)
```

Comment:
Maybe using a local variable would be nicer, since at the end it ends up setting this to None.